### PR TITLE
move the delete modal outside of the reply panel

### LINKED ISSUE #305: Delete modal for solution reply has blank text
When clicking the delete modal for a solution reply, it has blank text. This is probably due to the way the modal is nested inside the solution reply. Applying a color style should do the trick.

https://laravel.io/forum/06-04-2015-cipher-error

screen shot 2017-08-21 at 12 25 35 am

### PR DESCRIPTION
**Pull request by fetzi**
fixes #305

### CODE REVIEW

File: resources/views/forum/threads/show.blade.php [hunk 1]
```diff
@@ -112,6 +105,12 @@
                         @md($reply->body())
                     </div>
                 </div>
+                @include('_partials._delete_modal', [
```
Comment (driesvints) [inline resources/views/forum/threads/show.blade.php]: Add a white line above the inclue

File: resources/views/forum/threads/show.blade.php [hunk 2]
```diff
@@ -112,6 +105,12 @@
...
+                                'id' => "deleteReply{$reply->id()}",
+                                'route' => ['replies.delete', $reply->id()],
+                                'title' => 'Delete Reply',
+                                'body' => '<p>Are you sure you want to delete this reply? This cannot be undone.</p>',
+                            ])
```
Comment (driesvints) [inline resources/views/forum/threads/show.blade.php]: Indent this the way it was indented before.

File: resources/views/forum/threads/show.blade.php [hunk 3]
```diff
@@ -70,13 +70,6 @@
...
                             </div>
-
-                            @include('_partials._delete_modal', [
-                                'id' => "deleteReply{$reply->id()}",
-                                'route' => ['replies.delete', $reply->id()],
-                                'title' => 'Delete Reply',
-                                'body' => '<p>Are you sure you want to delete this reply? This cannot be undone.</p>',
-                            ])
                         @endcan
                     </div>
```
Comment (driesvints) [inline resources/views/forum/threads/show.blade.php]: Can we move the modal perhaps below this div? So it's closer to the hyperlink?

File: resources/views/forum/threads/show.blade.php (hunk 1 above)
Comment (fetzi) [reply resources/views/forum/threads/show.blade.php]: done

**Comment by driesvints** [comment]
Thanks! :)

**Pull request merged**